core/src/main/resources/hudson/model/Messages.properties

```diff
@@ -166,6 +166,9 @@ Hudson.AdministerPermission.Description=\
 Jenkins.Manage.Description=\
   This permission grants the ability to configure parts of the overall system configuration \
   that are not expected to have an impact on the overall stability and security of the system.
+Jenkins.SystemRead.Description=\
+  This permission grants read-only access to large parts of the overall system configuration. \
+  that are not expected to have an impact on the overall stability and security of the system.
```

Comment:
cut and paste error.  at least as I understand it SYSTEM_READ will allow you read access to everything?  (its like ADMINISTRATOR but without the ability to `save/apply`?